src/tainting/Dataflow_tainting.ml

```diff
@@ -575,6 +575,30 @@ let propagate_taint_to_label replace_labels label (taint : T.taint) =
   in
   { taint with orig = new_orig }
 
+let add_offset_to_poly_taint_in_st env lval st =
```

Comment:
Nit: the name is pretty opaque considering how many words it is. To me it seems like it should be closer to "resolve taints for getters"

Reply:
The name is clear to me right now, but perhaps I agree with you in X months time. So I'm renaming it to `resolve_poly_taint_for_java_getters` for now!